diff -urN openmortal-0.5-orig/src/sge_tt_text.cpp openmortal-0.5/src/sge_tt_text.cpp
--- openmortal-0.5-orig/src/sge_tt_text.cpp	2004-01-30 13:12:59.000000000 +0100
+++ openmortal-0.5/src/sge_tt_text.cpp	2004-02-03 00:04:29.000000000 +0100
@@ -37,9 +37,9 @@
 
 #ifndef _SGE_NOTTF
 #include <ft2build.h>
-#include <freetype/freetype.h>
-#include <freetype/ftoutln.h>
-#include <freetype/ttnameid.h>
+#include FT_FREETYPE_H
+#include FT_OUTLINE_H
+#include FT_TRUETYPE_IDS_H
 
 /* The structure used to hold glyph information (cached) */
 struct glyph {
